cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/common/utils.go

```diff
@@ -133,6 +133,17 @@ func NewLocalOpaqueSecret(ownerRepo types.NamespacedName) *apiv1.Secret {
 	}
 }
 
+// "Local" in the sense of no namespace is specified
+func NewLocalDockerConfigJsonSecret(ownerRepo types.NamespacedName) *apiv1.Secret {
+	return &apiv1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: ownerRepo.Name + "-",
```

Comment:
Perhaps we want to add some [common labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/) like _managed-by_ so that we can group/identify those resources more easily?

Reply:
That sounds good to me. A separate PR though, we make it consistent accross all plugins. This PR is about OCI support. Let me know and I will file an issue